### PR TITLE
Add Ludacris alerts to pmo

### PR DESCRIPTION
**Pull request by QuentinBisson**
## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

**Review by TheoBrigitte — APPROVED** [review]

**Review by MarcelMue — APPROVED** [review]

File: helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    cluster_type: "workload_cluster"
+  name: systemd.workload-cluster.rules
+spec:
+  groups:
+  - name: systemd
+    rules:
+    - alert: WorkloadClusterSystemdUnitFailed
+      annotations:
+        description: '{{`Systemd unit {{ $labels.name }} is failed on {{ $labels.instance }}.`}}'
+      expr: node_systemd_unit_state{cluster_type="workload_cluster",name!~"debug-tools.service|k8s-addons.service|sshd.+service|systemd-coredump.+service|systemd-networkd.service|update-engine-stub.service",state="failed"} == 1
```
Comment (TheoBrigitte) [inline helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml:16]: Removed `coreos-metadata-sshkeys@core.service`, and `locksmithd.service`. Is this correct @giantswarm/team-ludacris ?


original expr
```
node_systemd_unit_state{cluster_type="workload_cluster",name!~"coreos-metadata-sshkeys@core.service|debug-tools.service|k8s-addons.service|locksmithd.service|sshd.+service|systemd-coredump.+service|systemd-networkd.service|update-engine.service|update-engine-stub.service", state="failed"} == 1
```

File: helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      labels:
+        area: kaas
+        severity: page
+        team: ludacris
+        topic: infrastructure
+    - alert: WorkloadClusterDisabledSystemdUnitActive
+      annotations:
+        description: '{{`Disabled Systemd unit {{ $labels.name }} is active on {{ $labels.ip }}.`}}'
+        opsrecipe: disabled-systemd-unit-active/
+      expr: node_systemd_unit_state{cluster_type="workload_cluster",name=~"update-engine-stub.service",state="active"} == 1
```
Comment (TheoBrigitte) [inline helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml:38]: Removed `locksmithd.service`
Changed `update-engine.service` to `update-engine-stub.service`
Is this correct @giantswarm/team-ludacris ?


Original expr
```
node_systemd_unit_state{cluster_type="workload_cluster",name=~"locksmithd.service|update-engine.service", state="active"} == 1
```

Comment (MarcelMue) [reply helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml:38]: Let's please don't change these expressions right now. locksmithd needs to be disabled for sure

Comment (MarcelMue) [reply helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml:38]: I think changing any unit names in a follow up PR makes more sense to me.

Comment (TheoBrigitte) [reply helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml:38]: Ok, I'll revert this to the original expression and leave this for later then.

File: helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml (hunk 1 above)
Comment (TheoBrigitte) [reply helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml:16]: reverted to original expr

**Pull request merged**